### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by malcolmyanguci** [erikwijmans/Pointnet2_PyTorch#104]
Hi @erikwijmans, I am so new to pytorch-lighting. I did not find the loss function from the code of trainer. What is the loss function for the semantic segmentation? From other implementation for pointnet++, I found its just like `F.nll_loss()` but I still want to confirm if your version is using `F.nll_loss()` or you add the regularizer?

**Comment by erikwijmans** [comment]
The training step for all models is here: https://github.com/erikwijmans/Pointnet2_PyTorch/blob/master/pointnet2/models/pointnet2_ssg_cls.py#L125

I am using `F.cross_entropy` (which is softmax + nll_loss).

**Comment by malcolmyanguci** [comment]
thank you
